ingestion-beam/src/main/java/com/mozilla/telemetry/avro/BinaryRecordFormatter.java

```diff
@@ -8,7 +8,6 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.UncheckedIOException;
-
```

Comment:
I have gotten used to having this space between Java imports and others, so I was going to complain. But apparently this change is in line with Google Java Style. I'm generally in favor of sticking to whatever they recommend by default to avoid having to think too hard about it.

https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing